nidmviewer/viewer.py

```diff
@@ -202,17 +203,30 @@ def retrieve_nifti(peaks,retrieve,location_key):
     '''
     # Note: retrieve = True has not been tested!
     updated_peaks = dict()
-    for nidm,entries in peaks.iteritems():
-        if retrieve:
-            for e in range(len(entries)):
-                if location_key in entries[e]:
-                    brainmap = entries[e][location_key]
+    redirect_dict = dict()
+    for nidm,entries in peaks.iteritems(): 
```

Comment:
`peaks` parameter description is incomplete to understand what `nidm` is and items are.